src/main/java/eu/europeana/metis/sandbox/config/ConcurrencyJdbcConfig.java

```diff
@@ -0,0 +1,24 @@
+package eu.europeana.metis.sandbox.config;
+
+import javax.sql.DataSource;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.integration.jdbc.lock.DefaultLockRepository;
+import org.springframework.integration.jdbc.lock.JdbcLockRegistry;
+import org.springframework.integration.jdbc.lock.LockRepository;
+
+@Configuration
+public class ConcurrencyJdbcConfig {
+
+  @Bean
+  public DefaultLockRepository defaultLockRepository(DataSource dataSource) {
+    DefaultLockRepository defaultLockRepository = new DefaultLockRepository(dataSource);
```

Comment:
For the DefaultLockRepository and/or JdbcLockRegistry(Since I don't know where this would apply):
How about the TTL of the lock, or other similar parameters?
Shouldn't we have those as configuration parameters?
I think it can be useful in some situations where we'd need to adjust those.

In metis core we have configuration properties that control some values related to the locks that I think we should have control of.

Reply:
the previous comment disappeared. 
here is again, I have added the following parameters.
```
  @Value("${spring.integration.defaultlockrepository.prefix:integration.int_}")
  private String prefix;
  @Value("${spring.integration.defaultlockrepository.ttl:10000}")
  private int timeToLive;
  @Value("${spring.integration.defaultlockrepository.region:DEFAULT}")
  private String region;
```